Agda.tex

```diff
@@ -837,7 +838,7 @@ \section{Equational Proofs In Agda}
 proposition using equational reasoning within Agda's type system
 \cite{murray2022constructive}. 
 
-In section \ref{types}, we have seen how to define natural numbers and addition
+In section \ref{types}, we saw how to define the natural number and addition
```

Comment:
Actually, the original was correct (have seen -> saw is fine), but the other part's meaning has been broken by this change.